### PR TITLE
Added code of conduct and gitignore

### PR DESCRIPTION
**Pull request by GnikDroy**
Added a code of conduct from your sample to your repository.
Modified .gitignore to include class files and other unwanted files.

I have also modified README.md to include build options without Android Studio. This way you may get more contributors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,20 @@ _Follow these instructions to build and run the project.._
 6. Run the app. Click `Run > Run 'app'`. After the project builds you'll be
    prompted to build or launch an emulator.
 
+# Build with gradle
```
Comment (arshadkazmi42) [inline README.md]: Can you change it to ### as this will come under Getting Started and we want to keep the titles consistent everywhere.

**Comment by arshadkazmi42** [comment]
@GnikDroy Thank you for working on this.
I have mentioned a small change, with that change I think it will be ready to merge into master

**Comment by GnikDroy** [comment]
@arshadkazmi42 All right. Done.

**Comment by arshadkazmi42** [comment]
@GnikDroy Awesome. Merging it to master now 👍 
Congrats on your first contribution

**Pull request merged**